content/rule_book/internationals.md

```diff
@@ -1,62 +1,60 @@
+
+Any exceptions to this must be brought to the CQLT.
 
 ## Team Structure
 
-- Each district will be allowed to send as many teams as the district feels will be competitive and that they are financially able to send. These will be the district's International Teams.
-- Each team must have a coach and may also have an assistant coach.
+Each district will be allowed to send as many teams as the district feels will be competitive and that they are financially able to send. These will be the district's international teams.
+
+Each team must have a coach and may also have an assistant coach.
 
 ## Event Registration
 
-- All arrangements, promotion, and administration will be handled by the CQLT and LIFE Office.
-- Notification by each district of its intention to enter a team must be received by the CQLT, on a form supplied by the CQLT, no later than the date posted on the C&MA Quizzing website.
-- The names of the International Team, coaching staff, and all registration fees must be sent, when specified, to the LIFE Office for that quiz year.
-- District participation fees the current quiz year must be paid in order to register for the International competition. Any exceptions to this must be brought to the C&MA Quizzing Leadership Team (CQLT).
+All arrangements, promotion, and administration will be handled by the CQLT and the Life Office.
+
+Notification by each district of its intention to enter a team must be received by the CQLT, on a form supplied by the CQLT, no later than the date posted on the CMA Bible Quizzing website. The names of the IBQ team, its quizzers, and its coaching staff, and all registration fees must be sent, when specified, to the Life Office for that quiz season. District participation fees the current quiz season must be paid in order to register for the IBQ competition. Any exceptions to this must be brought to the CQLT.
 
 ## Selection of Officials
 
-- Officials for a sanctioned quiz may include a quizmaster, answer judge(s), and a scorekeeper(s).
-- The international quizmasters and officials will be appointed by the CQLT. To be considered for the CQLT appointment, all quizmasters and officials must:
-    - Be a regular church attender.
-    - Have three years of district quizmastering experience and be recommended by their district office or quiz committee.
-    - Go through an evaluation process established by the CQLT, to determine that each official is qualified to perform their specific role.
-- When deemed necessary, the quizmaster or any official may be replaced from their position during a meet by the C&MA Quizzing Leadership Team.
-- Team coaches whose teams are not involved in the quiz shall serve as answer judges and scorekeepers if needed.
+IBQ officials may include quizmasters, answer judges, scorekeepers, statisticians, and a meet director. IBQ officials will be appointed by the CQLT. To be considered for the CQLT appointment, all officials must:
```

Comment:
ALL officials must have 3 years of quizmastering experience? Not just quizmasters? Maybe I'm misunderstanding?

Reply:
The current rule book says all officials (QM, AJ, and SK) *must* have 3 years of district *quizmastering* experience, which doesn't make sense to me. For example, if someone has SKed for 10 years and never QMed or AJed, they'd make an excellent SK at IBQ (probably). Moreover, Meet Director(s) and Statistician(s) are not included in the list, and I'd say Statisticians need to be more experience than SKs, and MDs should be more experienced than QMs. In terms of application, these selection requirements aren't practically a "must"; they're really a "should". If the CQLT is struggling to find AJs and SKs, they're going to select local AJs and SKs regardless of quality or experience.

What we have in v1 is FE to what's in the current rule book, though it's not what practically happens unless we change "must" to "should".

My POV: Given that what's in v1 is FE to the current rule book, I think that's what we need to go with for v1. However, we should circle back to this for v2. I don't like reducing the standard to "should", but practical considerations may trump the ideal. Ultimately, this sounds like a question for Zach.

Reply:
Yeah, currently it's written poorly. I'm fine with what we have for FE. 

Down the road, I'd think a more robust description of Internationals officials selection is warranted (even if the description includes caveats for finances).

Reply:
I agree with all the above. Keep it as is for v1. I actually didn't _know_ the current rulebook said this, and yeah it makes no sense. Just wanted to make sure WE didn't introduce this bug...

We should definitely always enforce whatever rule is here, but I'm not sure we need this rule at all, particularly if we're already conceding that pragmatic concerns can supersede the rule. The CQLT should just just appoint whoever is best for the job. Naturally, anyone who makes that decision will factor in experience as one of several important considerations. David Swindler at Internet-ionals had 0.5 years of quizmastering experience and was in the top 10% of all Internationals QM's I've ever seen.

Reply:
Should is what I think makes the most sense with the current process. We almost always choose SKs who are local. STs should 100% have more experience than SKs but like Gryphon mentioned, they are listed. To my knowledge, in at least the past 5-6 years the SKs have not had 3 years of QM experience (if any QM experience at all)

I also think this statement is starting to become redundant as this line essentially trumps the previous line. 

> - Go through an evaluation process established by the CQLT, to determine that each official is qualified to perform their specific role

The other change that could occurs is that instead of saying quizmastering experience, it states "three years of experience in their IBQ official role."

But if all are in agreement with how it stands and making this an V2 issue, then let's make it a V2 issue

Reply:
Yesterday, I think I would have been happy to agree to keep it the same in v1, but being that today's a Monday, I'm second-guessing myself. I agree with everybody in this thread, but I'm struggling with the idea of knowingly leaving a "must" rule in place that we all know hasn't been and won't be followed. So, I propose the following: We leave everything the same for this section except we change "must" to "should". This would leave the in place what's in the current rule book but would soften it to reflect reality (and would allow for conformity to the rule book). And of course, we need to revisit this in v2. Not sure if there's a ticket filed for this yet or not.

Reply:
Totally fine with it. 

I think most everything surrounding Internationals officials selection is currently some combo of best practice and black box. So "should" reflects that. 

In the future it should be a bit more requirements-y, and public, but I think some amount of best practice and black box will always remain.

Reply:
dfd57c2

Reply:
Should makes sense to me. 

It's much closer to actual practice and clarifies the original intent (or at least what I would think the original intent was), though 3 years of matching official experience is probably the true actual intent.

Unless we used to require much more experienced officials tables